### DISCUSSION TITLE
2008/09/26/%E4%B8%BA%E4%BB%80%E4%B9%88%E5%9C%A8cln%E8%8A%82%E7%82%B9%E4%B8%AD%EF%BC%8C%E4%BC%9A%E8%87%AA%E5%8A%A8%E9%85%8D%E7%BD%AE%E6%9F%90%E4%BA%9Bdll%E5%87%BD%E6%95%B0%E7%9A%84%E5%8F%82%E6%95%B0%E4%BF%A1

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#245] [Announcements]
# 2008/09/26/%E4%B8%BA%E4%BB%80%E4%B9%88%E5%9C%A8cln%E8%8A%82%E7%82%B9%E4%B8%AD%EF%BC%8C%E4%BC%9A%E8%87%AA%E5%8A%A8%E9%85%8D%E7%BD%AE%E6%9F%90%E4%BA%9Bdll%E5%87%BD%E6%95%B0%E7%9A%84%E5%8F%82%E6%95%B0%E4%BF%A1

这是一个网友问我的问题，我开始也不了解。后来跟同事打听到了一些信息。

https://blog.qizhen.xyz/2008/09/26/%E4%B8%BA%E4%BB%80%E4%B9%88%E5%9C%A8cln%E8%8A%82%E7%82%B9%E4%B8%AD%EF%BC%8C%E4%BC%9A%E8%87%AA%E5%8A%A8%E9%85%8D%E7%BD%AE%E6%9F%90%E4%BA%9Bdll%E5%87%BD%E6%95%B0%E7%9A%84%E5%8F%82%E6%95%B0%E4%BF%A1

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Jing
2008年09月28日 09:40 编辑
嗯，差不多了。

回复
arrowpig
2008年09月26日 22:25 编辑
小牛牛，你的NET已经做好了？

回复
Jing
2008年09月26日 17:36 编辑
在VC中的Link属性中可以选择IDL文件插入。
建议使用.NET的dll（assembly），不需要配置参数。

回复
